### DISCUSSION TITLE
How to always save the credit card for later usage in the payment sheet?

### DISCUSSION THREAD

**Discussion by joeloudjinz** [flutter-stripe/flutter_stripe#345] [Q&A] [answered]
Is it possible to remove the checkbox that says: "Save my credit card information" and save the cards of the user with the payment sheet?

**Comment by remonh87** [comment, marked as answer]
No, as far as I know. you need to ask the user if they want to save their info for the next time.  From privacy and data regulations I can also imagine the way Stripe designed it makes sense (we have no control over this component).
